geomstats/geometry/poincare_ball.py

```diff
@@ -60,6 +66,132 @@ def belongs(self, point, tolerance=TOLERANCE):
         """
         return gs.sum(point**2, axis=-1) < (1 - tolerance)
 
+    @staticmethod
+    def pdf(data, means, variances, norm_func, metric):
+        """Return the probability density function.
```

Comment:
Maybe call this `gmm_pdf`? and: Returns the probability density function of the Gaussian Mixture Model?